bad_code.java

```diff
@@ -8,14 +8,15 @@ public static void main(String[] args) {
 }
 
 class ParentClass {
```

Comment:
Consider adding comments to describe the purpose and functionality of the `ParentClass` class. [medium]